### DISCUSSION TITLE
how to insert format date 'yyyy-mm-dd' for text input type date

### DISCUSSION THREAD

**Discussion by julio3266** [react-hook-form/react-hook-form#8548] [General]
How do I insert a format of 'yyyy-mm-dd' to a com input when running onSubmit?
<img width="305" alt="image" src="https://user-images.githubusercontent.com/52897217/175007590-2b458f46-69a2-4c2e-bfc2-7f3c1ac805d4.png">

**Comment by bluebill1049** [comment]
you may want to do inside onSubmit function or potentially use a mask input.

**Reply by julio3266** [reply]
it would only be when sending (onSubmit)
